docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+```
+CORE_PEER_LOCALMSPID="Org1MSP"
+CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
+CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+CORE_PEER_ADDRESS=localhost:7051
+```
+
+You can now approve the chaincode definition as Org1.
+```
+peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name fabcar --version 1.0 --init-required --package-id $CC_PACKAGE_ID --sequence 1 --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+We now have the majority we need to deploy the Fabcar the chaincode to the channel. While only a majority of organizations need to approve a chaincode definition (with the default policies) all organizations need to approve a chaincode definition to start the chaincode on their peers. As a result, it is recommended that all channel members approve a chaincode before committing the chaincode definition.
+
+## Committing the chaincode definition to the channel
+
+After a sufficient number of organizations have approved a chaincode definition, one organization can commit the chaincode definition to the channel. If a majority of channel members have approved have approved the same chaincode definition, the commit transaction will be successful and the parameters agreed in the chaincode definition will be implemented on the channel.
+
+You can use the `peer lifecycle chaincode checkcommitreadiness` command to check whether channel members have approved the same chaincode definition. Use the following command to check if Org1 and Org2 have approved the chaincode definition for Fabcar on the channel `mychannel`. The flags used for the `checkcommitreadiness` command are identical to the flags used to approve a chaincode for your organization. However, you do not need to include the `--package-id` flag.
+```
+peer lifecycle chaincode checkcommitreadiness --channelID mychannel --name fabcar --version 1.0 --init-required --sequence 1 --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --output json
+```
+
+The command will produce a JSON map that display if a channel member member has approved the parameters that were specified in the `checkcommitreadiness` command:
+```json
+    {
+            "Approvals": {
+                    "Org1MSP": true,
+                    "Org2MSP": true
+            }
+    }
+```
+
+Since both channel members have approved the same parameters, the chaincode definition can be committed to the channel. Issue the following command to commit the chaincode definition to the channel as the Org2 admin:
+```
```

Comment:
Would this be since both organizations have approved vs. both members?

Reply:
Changed to: "Since both organizations that are members of the channel have approved the same parameters"